sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -121,6 +125,16 @@ public static RequestContent Create(object serializable, JsonPropertyNames prope
             ObjectSerializer serializer = new JsonObjectSerializer(serializerOptions);
             return Create(serializer.Serialize(serializable));
         }
+        /// <summary>
+        /// Creates an instance of <see cref="RequestContent"/> that wraps a serialized version of an object.
+        /// </summary>
+        /// <param name="model">The model to serialize.</param>
+        /// <param name="options">The format to use for property names in the serialized content.</param>
+        /// <returns>An instance of <see cref="RequestContent"/> that wraps a serialized version of the model.</returns>
+        public static RequestContent Create<T>(IPersistableModel<T> model, ModelReaderWriterOptions options)
```

Comment:
This will be added in Azure.Core 2.0.  Do we need it before that ships?

Reply:
@annelo-msft This method is needed for multipart feature. I added there for current prototyping. When Azure.Core 2.0 released and I will remove the duplicate one. Thanks

And I think we can provide a default value `ModelReaderWriterOptions.Json` for `options` parameter.